ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1231,4 +1231,45 @@ year = Year
 year.plural = Years
 years = years
 yes = Yes
-yesterday = yesterday
\ No newline at end of file
+yesterday = yesterday
+permission.description.can_access_gateway_api = Allowed to access APIs to act as an SMS gateway.
+permission.description.can_bulk_delete_reports = Allowed to use bulk delete functions to delete multiple reports.
+permission.description.can_configure = Allowed to configure all the application configurations.
+permission.description.can_create_people = Allowed to create new people.
+permission.description.can_create_places = Allowed to create new places.
+permission.description.can_create_records = Allowed to access the APIs to create reports.
+permission.description.can_create_users =  Allowed to create users.
+permission.description.can_delete_contacts = Allowed to delete people and places.
+permission.description.can_delete_messages = Allowed to delete messages.
+permission.description.can_delete_reports = Allowed to delete reports.
+permission.description.can_delete_users = Allowed to delete users.
+permission.description.can_edit = Allowed to create and edit documents in the database.
```

Comment:
I find this guy pretty confusing and often misunderstood. Maybe adding a statement like "If this is false, the user can create documents on their phone but they will not sync to the server."

Reply:
I don't want to complicate things with concepts like "sync". The important part IMO is that from an admins perspective this user cannot create/update docs in the CouchDB database. I'm not sure how to improve this...